### ISSUE TITLE
Jenkins TV board for accuracy and speed of h2o4gpu

### ISSUE THREAD

**Issue by pseudotensor** [h2oai/h2o4gpu#22]
For all of glm vs. h2o-3, kmeans vs sklearn/Intel daal, xgboost vs. lightgbm
